resources/qml/ActionPanel/OutputProcessWidget.qml

```diff
@@ -128,7 +128,7 @@ Column
             tooltip: text
             fixedWidthMode: true
 
-            toolTipContentAlignment: Cura.ToolTip.ContentAlignment.AlignLeft
+            toolTipContentAlignment: Text.AlignLeft
```

Comment:
After wrecking my brain a bit more about this, I've found a more elegant solution! The issue that I have with this is that there are more alignment options in text (and of course that using the Text alignment to align content also feels weird).

So the solution that I've found is that adding a Enums.qml, that contains the enum that you need does the trick.

```
import QtQuick 2.15
import QtQuick.Controls 2.10

Item
{
    enum ContentAlignment
    {
        AlignLeft,
        AlignRight
    }
}
```

This needs to be exposed in the qmldir

`Enums                   1.5 Enums.qml`

The orientation can then be set as you would expect:
`property int contentAlignment: UM.Enums.ContentAlignment.AlignLeft`

Although I don't know why this didn't fail when it was in the cura namespace, i suspect that it has to do with it mirroring the exact name of the controls tooltip. When I renamed the tootlip qml to tooltipp, it did work.